src/main/java/lotto/domain/lotto/LottoTicket.java

```diff
@@ -0,0 +1,35 @@
+package lotto.domain.lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lotto.domain.Rank;
+import lotto.domain.lotto.LottoLine;
+import lotto.domain.lotto.LottoNumber;
+
+public class LottoTicket {
+
+    private final List<LottoLine> lottoLines;
+
+    public LottoTicket(List<LottoLine> lottoLines) {
+        this.lottoLines = lottoLines;
+    }
+
+    public int getLength() {
+        return lottoLines.size();
+    }
+
+    public List<Rank> matchLottoLines(List<LottoNumber> answerLottoNumbers,
+                                      LottoNumber bonusNumber) {
+        List<Rank> ranks = new ArrayList<>();
+        for (LottoLine lottoLine : lottoLines) {
+            ranks.add(lottoLine
+                    .matchLottoNumbers(lottoLine.getValues(), bonusNumber, answerLottoNumbers));
```

Comment:
줄바꿈을 하고 있는 이유가 있을까요? :)
```suggestion
    public List<Rank> matchLottoLines(List<LottoNumber> answerLottoNumbers, LottoNumber bonusNumber) {
        List<Rank> ranks = new ArrayList<>();
        for (LottoLine lottoLine : lottoLines) {
            ranks.add(lottoLine.matchLottoNumbers(lottoLine.getValues(), bonusNumber, answerLottoNumbers));
```

Reply:
별다른 이유는 없습니다 ㅜ 컨벤션 실수 였던것 같습니다